crates/ruff/src/registry.rs

```diff
@@ -443,83 +444,83 @@ ruff_macros::register_rules!(
+    rules::pandas_vet::rules::PandasUseOfDotAt,
+    rules::pandas_vet::rules::PandasUseOfDotIat,
+    rules::pandas_vet::rules::PandasUseOfDotPivotOrUnstack,
+    rules::pandas_vet::rules::PandasUseOfDotValues,
+    rules::pandas_vet::rules::PandasUseOfDotReadTable,
+    rules::pandas_vet::rules::PandasUseOfDotStack,
+    rules::pandas_vet::rules::PandasUseOfPdMerge,
+    rules::pandas_vet::rules::PandasDfVariableName,
     // flake8-errmsg
     rules::flake8_errmsg::rules::RawStringInException,
     rules::flake8_errmsg::rules::FStringInException,
     rules::flake8_errmsg::rules::DotFormatInException,
     // flake8-pyi
-    rules::flake8_pyi::rules::PrefixTypeParams,
+    rules::flake8_pyi::rules::UnprefixedTypeParam,
     rules::flake8_pyi::rules::UnrecognizedPlatformCheck,
     rules::flake8_pyi::rules::UnrecognizedPlatformName,
     rules::flake8_pyi::rules::PassStatementStubBody,
     rules::flake8_pyi::rules::NonEmptyStubBody,
     rules::flake8_pyi::rules::DocstringInStub,
-    rules::flake8_pyi::rules::TypedArgumentSimpleDefaults,
-    rules::flake8_pyi::rules::ArgumentSimpleDefaults,
+    rules::flake8_pyi::rules::TypedArgumentDefaultInStub,
+    rules::flake8_pyi::rules::ArgumentDefaultInStub,
     // flake8-pytest-style
-    rules::flake8_pytest_style::rules::IncorrectFixtureParenthesesStyle,
-    rules::flake8_pytest_style::rules::FixturePositionalArgs,
-    rules::flake8_pytest_style::rules::ExtraneousScopeFunction,
-    rules::flake8_pytest_style::rules::MissingFixtureNameUnderscore,
-    rules::flake8_pytest_style::rules::IncorrectFixtureNameUnderscore,
-    rules::flake8_pytest_style::rules::ParametrizeNamesWrongType,
-    rules::flake8_pytest_style::rules::ParametrizeValuesWrongType,
-    rules::flake8_pytest_style::rules::PatchWithLambda,
-    rules::flake8_pytest_style::rules::UnittestAssertion,
-    rules::flake8_pytest_style::rules::RaisesWithoutException,
-    rules::flake8_pytest_style::rules::RaisesTooBroad,
-    rules::flake8_pytest_style::rules::RaisesWithMultipleStatements,
-    rules::flake8_pytest_style::rules::IncorrectPytestImport,
-    rules::flake8_pytest_style::rules::AssertAlwaysFalse,
-    rules::flake8_pytest_style::rules::FailWithoutMessage,
-    rules::flake8_pytest_style::rules::AssertInExcept,
-    rules::flake8_pytest_style::rules::CompositeAssertion,
-    rules::flake8_pytest_style::rules::FixtureParamWithoutValue,
-    rules::flake8_pytest_style::rules::DeprecatedYieldFixture,
-    rules::flake8_pytest_style::rules::FixtureFinalizerCallback,
-    rules::flake8_pytest_style::rules::UselessYieldFixture,
-    rules::flake8_pytest_style::rules::IncorrectMarkParenthesesStyle,
-    rules::flake8_pytest_style::rules::UnnecessaryAsyncioMarkOnFixture,
-    rules::flake8_pytest_style::rules::ErroneousUseFixturesOnFixture,
-    rules::flake8_pytest_style::rules::UseFixturesWithoutParameters,
+    rules::flake8_pytest_style::rules::PytestIncorrectFixtureParenthesesStyle,
```

Comment:
Some of these are very long unfortunately.

Reply:
I think `PytestFixtureIncorrectParenthesesStyle` would be slightly better (though not shorter). I don't consider the length to be a problem. We have autocompletion in the VS code extension as well as autocompletion for the CLI.

Sidenote: We don't support the [`pytest-fixture-no-parentheses` config setting](https://github.com/m-burst/flake8-pytest-style/blob/v1.7.2/docs/rules/PT001.md), do we?